### ISSUE TITLE
Does not work with Webpack 5

### ISSUE THREAD

**Issue by kirill-konshin** [Urthen/case-sensitive-paths-webpack-plugin#40]
Here's the error:

```
/Users/xxx/node_modules/case-sensitive-paths-webpack-plugin/index.js:163
      let pathName = data.resource.split('?')[0];
                                   ^
TypeError: Cannot read property 'split' of undefined
    at primeCache (/Users/xxx/node_modules/case-sensitive-paths-webpack-plugin/index.js:163:36)
    at that.getFilenamesInDir (/Users/xxx/node_modules/case-sensitive-paths-webpack-plugin/index.js:130:5)
    at Array.fs.readdir (/Users/xxx/node_modules/case-sensitive-paths-webpack-plugin/index.js:66:5)
    at Storage.finished (/Users/xxx/node_modules/webpack/node_modules/enhanced-resolve/lib/CachedInputFileSystem.js:55:16)
    at provider (/Users/xxx/node_modules/webpack/node_modules/enhanced-resolve/lib/CachedInputFileSystem.js:91:9)
    at go$readdir$cb (/Users/xxx/node_modules/graceful-fs/graceful-fs.js:162:14)
    at FSReqCallback.args [as oncomplete] (fs.js:145:20)
```

Most likely due to API changes.

**Comment by luoxue-victor** [comment]
When does the author plan to fix it?

**Comment by Urthen** [comment]
Just published 2.3.0 which should fix support in Webpack 5, at least as of `5.0.0-beta.11`.

Sorry this sat so long - I was hoping Webpack 5 would release (or at least get closer) to minimize risk of future breaking changes before saying "it's supported now"; then just dropped of my radar.
